packages/plugin-ext/src/plugin/plugin-manager.ts

```diff
@@ -185,7 +188,10 @@ export class PluginManagerExtImpl implements PluginManagerExt, PluginManager {
         this.registry.set(plugin.model.id, plugin);
         if (plugin.pluginPath && Array.isArray(plugin.rawModel.activationEvents)) {
             const activation = async () => {
+                const title = `Loading ${plugin.model.displayName || plugin.model.name}`;
+                const id = await this.notificationMain.$startProgress({title, location: 'window'});
```

Comment:
please use Theia or VS Code that formatting on save works

Reply:
fixed